### DISCUSSION TITLE
Login With GitHub

### DISCUSSION THREAD

**Discussion by unrealapex** [monkeytypegame/monkeytype#1495] [Ideas] [answered]
It would be nice to be able to login with GitHub on Monkeytype.

**Comment by Otard95** [comment, marked as answer]
As the app using Firebase authentication. it should at lest not be to difficult to implement as its an option for sign-in providers. Though I wonder how many using the app, also uses GitHub. I think that might be a good question to answer first(though I personally like the idea).

**Reply by unrealapex** [reply]
I would assume about 3,000 based on the stars of this project.

**Reply by unrealapex** [reply]
> As the app using Firebase authentication. it should at lest not be to difficult to implement as its an option for sign-in providers. 

Yep, https://firebase.google.com/docs/auth/web/github-auth

**Comment by tusqasi** [comment]
I would also lilke this
